frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -145,15 +152,35 @@ export const monitoringAlertRows = (
   return rows;
 };
 
-export const alertFilters = [
-  {
-    filterGroupName: 'Alert State',
-    type: 'alert-state',
-    reducer: alertState,
-    items: [...alertsRowFilters[0].items, ...[{ id: 'inactive', title: 'Inactive' }]],
-  },
-  severityRowFilter,
-];
+export const getFilteredRows = (_filters, rowFilters, objects) => {
+  if (_.isEmpty(_filters)) {
+    return objects;
+  }
+
+  const allTableFilters = getAllTableFilters(rowFilters);
+  let filteredObjects = objects;
+  _.each(_filters, (value, name) => {
+    const filter = allTableFilters[name];
+    if (_.isFunction(filter)) {
+      if (name === 'alert-state') {
+        if (!_.isEmpty(value.selected)) {
+          filteredObjects = _.filter(
+            filteredObjects,
+            (o) =>
+              _.filter(o.alerts, (a) => filter(value, a)).length > 0 ||
```

Comment:
```suggestion
              _.some(o.alerts, (a) => filter(value, a)) ||
```

wdyt?

Reply:
This function is not required. So, removed it.